src/ServiceControl.Monitoring/Infrastructure/EndpointTracker.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace ServiceControl.Monitoring.Infrastructure
+{
+    using System.Threading.Tasks;
+    using Metrics.Raw;
+    using NServiceBus;
+    using NServiceBus.Metrics;
+
+    public class EndpointTracker : IHandleMessages<MetricReport>, IHandleMessages<LongValueOccurrences>, IHandleMessages<Occurrences>
```

Comment:
@mikeminutillo point taken. Can we make it a separate PR though? I spent some time yesterday trying to make it work but for some reason when switching to `IHandleMessages<IMessage>` or `IHandleMessages<object>` results in `AutoFac` exception - it complains that it cannot find dependencies for the constructor :/.